module/VuFind/src/VuFind/Section/SectionServiceInterface.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+/**
+ * Section service interface.
+ *
+ * PHP version 8
+ *
+ * Copyright (C) The National Library of Finland 2025.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see
+ * <https://www.gnu.org/licenses/>.
+ *
+ * @category VuFind
+ * @package  Section
+ * @author   Aleksi Peebles <aleksi.peebles@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+
+namespace VuFind\Section;
+
+use VuFind\Config\Feature\ConfigSettingPropertiesInterface;
+
+/**
+ * Section service interface.
+ *
+ * @category VuFind
+ * @package  Section
+ * @author   Aleksi Peebles <aleksi.peebles@helsinki.fi>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Site
+ */
+interface SectionServiceInterface
+{
+    public const DEFAULT_CONFIG_FILE = 'Sections.yaml';
+
+    /**
+     * Get section.
+     *
+     * @param string       $key    Section key in configuration
+     * @param array|string $config Configuration or configuration file name
+     *                             (optional)
```

Comment:
This can be unwrapped:

```suggestion
     * @param array|string $config Configuration or configuration file name (optional)
```